tools/cloud-build/daily-tests/ansible_playbooks/test-slurm-v6-tpu.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Assert variables are defined
+  ansible.builtin.assert:
+    that:
+    - custom_vars.partitions is defined
+
+- name: Get partition info
+  ansible.builtin.command: sinfo --format='%P' --noheader
+  changed_when: False
+  register: partition_output
+  retries: 10
+  delay: 12
+  until: partition_output.rc == 0
+
+- name: Count Slurm nodes
+  ansible.builtin.shell:
+    sinfo -t 'IDLE&POWERED_DOWN' --noheader --format "%n"
+  args:
+    executable: /bin/bash
+  changed_when: False
+  register: initial_node_count
+
+- name: Check partitions exist
+  ansible.builtin.fail:
+    msg: Test Check Partitions failed
+  when: item not in partition_output.stdout
+  loop: "{{ custom_vars.partitions }}"
+
+- name: Run JAX verification
+  register: crd_status
+  failed_when: crd_status.rc != 0
+  ansible.builtin.command: |
```

Comment:
Do we need to add `set -e` so `ansible.builtin.command` fails if `srun` fails?

Reply:
I added return code in the python script. This would return error code 1 in case of failure and would fail the build.